labs/iot/gps-lab/client/code/raspberry-pi-gps-tracker.py

```diff
@@ -0,0 +1,95 @@
+import time
```

Comment:
Can we have more comments explaining the code in detail - we don't want to make any assumptions about the users knowledge level or comfort with Python - better to have more comments than too little.